spec/factories/listings.rb

```diff
@@ -1,7 +1,6 @@
 FactoryBot.define do
   factory :listing do
     location
-    service_area
```

Comment:
per discussion, we still need this

Reply:
@maebeale If it's OK with you, I want to merge this in first. Then we can add this back to the factory once the db supports it